### DISCUSSION TITLE
OpenSearch support

### DISCUSSION THREAD

**Discussion by mikewink** [javascript-tutorial/en.javascript.info#2916] [Ideas]
Hello,

would it be possible to add OpenSearch support for this great project? Details about the super simple implementation can be found over on mdn.

https://developer.mozilla.org/en-US/docs/Web/OpenSearch

I would love to be able to add your site as a custom search so I can have a direct reference when I have to look something up. I'm sure others would find it very helpful too. 

Best regards

Mike

**Comment by Rnbsov** [comment]


**Reply by mikewink** [reply]
<img width="785" alt="Screenshot 2022-03-20 at 15 18 46" src="https://user-images.githubusercontent.com/85063626/159166884-63bf5f49-f501-4a0c-88cc-4133aca828a0.png">

I mean searching in Firefox for example and getting a chance to tab and select on of the targets, here the default ones like Google or Bing. But you can add your own Icons here if the site provides OpenSearch and therefore search directly. 

Not sure if your screenshot means the same, could be only suggestions based on former searches.

**Reply by ShahriarKh** [reply]
Another Example: MDN
![image](https://user-images.githubusercontent.com/31452340/161374746-7d721fce-3b4a-4b88-8ab4-a753a63c83ae.png)

**Comment by ShahriarKh** [comment]
Nice idea.

**Comment by iliakan** [comment]
What should I add, exactly? 😀

Tell me, that would speed up the process greatly 👍
